aten/src/ATen/native/LinearAlgebraUtils.h

```diff
@@ -12,15 +12,21 @@
 
 namespace at { namespace native {
 
+enum class TransposeType {
```

Comment:
Could you please add a comment that this enum is used for interfacing with different BLAS-like libraries?